### ISSUE TITLE
Icon for editor description box.

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot-proposals#4558]
### Describe the project you are working on

Well, there is no way to know at a glance which nodes have information defined in the "Editor description" box, so I suggest the possibility of showing an icon, for example, a book icon, to indicate that that node has information in its "Editor description" box.

I'm creating some levels and putting some information on some nodes in this box but afterwards, I don't know which nodes have information located there because godot doesn't show any icon or any way to identify nodes that have information defined in their "Editor description" box. 

Example book icon
![Captura](https://user-images.githubusercontent.com/19652075/168994865-9ebd12ba-f6b9-4b2c-9634-f828ebcc6fa0.PNG)

### Describe the problem or limitation you are having in your project

Easily identify which nodes have information in their "Editor description" box.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

An icon will be displayed on the node to indicate that it has information in its "Editor description" box.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

...

### If this enhancement will not be used often, can it be worked around with a few lines of script?

IDK

### Is there a reason why this should be core and not an add-on in the asset library?

I don't know if this can be fixed with a script but I don't think so.

**Comment by Calinou** [comment]
I like the idea in principle, but I'm concerned about having too many icons displayed in the scene tree dock.

**Comment by Zireael07** [comment]
Re-use the warning (I mean both icon and code for tooltip) then?

**Comment by YuriSizov** [comment]
I wouldn't reuse the warning icon just for node description. Besides, a node can have both a configuration warning and a description.

**Comment by SilencedPerson** [comment]
not necessarily, grey or white warning icon could be used to signify description and turn it yellow if there is a warning present.
hovering the cursor would display both, the warning and the description.



> I like the idea in principle, but I'm concerned about having too many icons displayed in the scene tree dock. This can be problematic when using a narrow dock layout.

we are reaching a point where the icons need ability to be turned off and on in the display.

**Comment by kleonc** [comment]
Signals/groups buttons are shown as one when both are present:
![Godot_v3 4 4-stable_win64_dUcMli9Xyh](https://user-images.githubusercontent.com/9283098/169145745-e35a9437-e583-4468-a014-2d0e4d1139c0.png)
![dvtSM1hinD](https://user-images.githubusercontent.com/9283098/169145750-a7238663-3d8c-4dc6-b336-2dd51e138ad2.png)
![RdI4ohLGSN](https://user-images.githubusercontent.com/9283098/169145749-445b0444-80aa-4bf4-b0e7-6b867f3a7369.png)

Maybe the same could be done for description/warning or description/scene unique name? :thinking: However these aren't related to each other (while signals and groups are editable in the same dock).

**Comment by Calinou** [comment]
> we are reaching a point where the icons need ability to be turned off and on in the display.

See https://github.com/godotengine/godot-proposals/issues/1608. I'm not convinced by this approach, as it suffers from poor discoverability and requires additional tweaking from the user to be effective.

**Comment by SilencedPerson** [comment]
> See #1608. I'm not convinced by this approach, as it suffers from poor discoverability and requires additional tweaking from the user to be effective. See the comment on Blender in that proposal 🙂

I do not like how blender handles them either, ideally they all should be on by default.
What about hiding the icons (except the warnings) when the dock width passes certain threshold?

**Comment by Calinou** [comment]
> What about hiding the icons (except the warnings) when the dock width passes certain threshold?

I'm not fond of hiding functionality entirely if your dock is too narrow.

**Comment by SilencedPerson** [comment]
Then the only option left that I can think of is to reflow the icons under the name of the node when the docks gets too small.
What else is there?

**Comment by Jummit** [comment]
Some more ideas which I don't necessarily like, but which are possible:

* Add a little exclamation mark above the node icon. This would require changing how Tree works, and would probably not look very convincing.
* Mark the node in the tree in some other way, like changing the font color, background or icon. Nodes with descriptions are probably important anyway, so I don't think it would be bad to highlight them.
* Show a popup when the scene is first opened, which shows the editor information of the nodes which have it. This would of course need an editor option to turn it off.

**Comment by Mickeon** [comment]
I wonder if it would be possible to display the Description if it exists in a way that "pops" for users exploring the Scene Tree in need for information, while not being **potentially** cluttering (e.g. the icon of this proposal). It's worth considering that Nodes with description are highly likely to display other icons already to complement their importance (Scene Unique, Script, Group, Signals...).

Currently, the editor description is mostly noticeable if you look for it in the Inspector. Alternatively, hovering on a Node to display and checking if it even exists... is typically a slow chore in the long run.
![image](https://user-images.githubusercontent.com/66727710/169660829-17603e5c-ee91-4075-8681-e10ccb4b27c1.png)

**Comment by aXu-AP** [comment]
This would be very good for making demos/tutorials. A way to explain how the scene works without adding extra labels or external readme is appreciated. Currently descriptions are quite obscure and I barely see them being used. As for the icon, I think an i inside a circle would be very recognisable. ℹ️
